packages/dds/sequence/src/revertibles.ts

```diff
@@ -158,17 +158,25 @@ export function appendChangeIntervalToRevertibles(
 	revertibles: SharedStringRevertible[],
 ) {
 	const startSeg = previousInterval.start.getSegment() as SharedStringSegment;
+	const startType =
```

Comment:
Add a comment here to explain that we can't use StayOnRemove in removed segments. And that it's okay for revertibles because the remove range revertible already handles the old position within the removed segment.